Dockerfile

```diff
@@ -1,40 +1,28 @@
 FROM node:10.13-alpine
 
-# Install required libraries
-RUN apk --no-cache add bash curl git \
-  make gcc g++ python \
-  libsecret-dev glib-dev
-
-# Set up deploy user and working directory
-RUN adduser -D -g '' deploy
-RUN mkdir -p /app
-RUN chown deploy:deploy /app
-
-# Set up dumb-init
-ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 /usr/local/bin/dumb-init
-RUN chown deploy:deploy /usr/local/bin/dumb-init
-RUN chmod +x /usr/local/bin/dumb-init
-
-# Copy required libs and files to /app
-ADD . /app
-ADD package.json /app
-ADD yarn.lock /app
 WORKDIR /app
 
-# Set up node_modules
-RUN yarn cache clean && yarn install
+RUN apk --no-cache --quiet add \
+      bash \
+      build-base \
+      curl \
+      dumb-init \
+      git \
+      python && \
+      adduser -D -g '' deploy
 
```

Comment:
I think a comment would be helpful here, perhaps another on line #19 to clarify why we copy twice.